packages/mobile/src/components/Files/ChatFileOverview.tsx

```diff
@@ -192,7 +231,7 @@ export const ChatFileOverview = memo(
             backgroundColor: 'pink',
           }}
         >
-          <Header title={title || ''} headerLeft={headerLeft} />
+          <Header title={'Send Message'} headerLeft={headerLeft} />
```

Comment:
As the ChatFIleOverview is used both for sharing and within a single conversation, perhaps keep the title of that conversation visible.. Or users would lose context